src/com/lftechnology/java/training/sanish/numbergame/GuessNumberGame.java

```diff
@@ -0,0 +1,137 @@
+package com.lftechnology.java.training.sanish.numbergame;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Guess number program in between 1 - 20
+ * 
+ * @author Sanish Maharjan <sanishmaharjan@lftechnology.com>
+ */
+public class GuessNumberGame {
+	private static final Logger LOGGER = Logger
+			.getLogger(GuessNumberGame.class.getName());
+	private static final int MIN_RANDOM_NUM = 1;
+	private static final int MAX_RANDOM_NUM = 20;
+
+	private int randomNum;
+	private int guessNum;
+	private int guessCount = 0;
+
+	/**
+	 * Constructor
+	 * 
+	 * @author Sanish Maharjan <sanishmaharjan@lftechnology.com>
+	 */
+	public GuessNumberGame() {
+		int randNum = (int) (Math.random() * MAX_RANDOM_NUM) + MIN_RANDOM_NUM;
```

Comment:
You can use java.util.Random here instead of Math.random.
http://www.tutorialspoint.com/java/util/random_nextint_inc_exc.htm